### PR TITLE
User branch

### PR DESCRIPTION
**Pull request by Nikhil31Lathiya**
### Proposed Changes
- validation changes in date field
- swagger implementation in src/lib/swagger/swagger.json package

### CODE REVIEW

File: src/lib/swagger/swagger.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "openapi": "3.0.3",
     "info": {
       "title": "OZA-LAB APIs",
-      "version": "1.0.0",
+      "version": "3.0.3",
```
Comment (RutuThakkar510) [inline src/lib/swagger/swagger.json:5]: change the version to 1.0.0

File: src/lib/swagger/swagger.json [hunk 2]
```diff
@@ -419,11 +419,16 @@
...
+                "role",
+                "title",
                 "firstName",
-                "lastName",
+                "address",
                 "email",
-                "creationDate",
-                "updateDate"
+                "gender",
+                "dateOfBirth",
```
Comment (RutuThakkar510) [inline src/lib/swagger/swagger.json:428]: change dateOfBirth to dob 
use same field name as in db.

File: src/lib/swagger/swagger.json [hunk 3]
```diff
@@ -448,21 +462,42 @@
...
+            },
+            "gender":{
+              "type": "string",
+              "example":"[male] [female]"
+            },
+            "title":{
+              "type": "string",
+              "example":"[mr] [ms] [baby] [master] [dr] "
+            },
+            "dateOfBirth":{
```
Comment (RutuThakkar510) [inline src/lib/swagger/swagger.json:490]: change it to dob

File: src/utils/validations.js [hunk 4]
```diff
@@ -1,11 +1,19 @@
 import Joi from 'joi'
+// dob: Joi.date().format('YYYY-MM-DD').options({ convert: false }),
```
Comment (RutuThakkar510) [inline src/utils/validations.js:2]: remove commented code

File: src/lib/swagger/swagger.json (hunk 1 above)
Comment (Nikhil31Lathiya) [reply src/lib/swagger/swagger.json:5]: done

File: src/lib/swagger/swagger.json (hunk 2 above)
Comment (Nikhil31Lathiya) [reply src/lib/swagger/swagger.json:428]: done

File: src/lib/swagger/swagger.json (hunk 3 above)
Comment (Nikhil31Lathiya) [reply src/lib/swagger/swagger.json:490]: done

**Review by Nikhil31Lathiya — COMMENTED** [review]
done

**Pull request merged**